coref/coref_model.py

```diff
@@ -389,7 +387,8 @@ def _bertify(self, doc: Doc) -> torch.Tensor:
         return out[subword_mask_tensor]
 
     def _build_model(self):
-        self.bert, self.tokenizer = bert.load_bert(self.config)
+        self.bert = self.config.model_bank.encoder
```

Comment:
This is related to my comment about having a model stored in config and this https://app.asana.com/0/1203072224624647/1203086714336444/f